include/posix/pthread_sched.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
```

Comment:
There's no need to include `stdint.h` here: this header file is not using any definition from this header.  Add the `#include` directive to the implementation file.  Also, please add an include guard.